indexers/gemini-3h/staking/src/mappings/mappingHandlers.ts

```diff
@@ -1,33 +1,191 @@
 import { SubstrateEvent, SubstrateExtrinsic } from "@subql/types";
 import assert from "assert";
 import {
-  Account,
   Bundle,
   BundleAuthor,
   Deposit,
-  DepositStatus,
-  Domain,
   DomainBlock,
-  DomainEpoch,
-  DomainRuntime,
   Nominator,
+  Withdrawal,
+} from "../types";
+import * as db from "./db";
+import {
+  DepositStatus,
+  DomainRuntime,
   NominatorPendingAction,
   NominatorStatus,
-  Operator,
   OperatorPendingAction,
   OperatorStatus,
-  Reward,
-  Stats,
-  StatsPerAccount,
-  StatsPerDomain,
-  StatsPerNominator,
-  StatsPerOperator,
-  Withdrawal,
   WithdrawalStatus,
-} from "../types";
+} from "./models";
+import { capitalizeFirstLetter, stringify } from "./utils";
 
 export const SHARES_CALCULATION_MULTIPLIER = BigInt(1000000000000);
 
+export interface TBundle {
```

Comment:
Wouldn't be good to have these types in a separate?

Reply:
You are right 👍